src/PokemonContainer.js

```diff
@@ -3,9 +3,9 @@ import Card from "./Card";
 function PokemonContainer({ pokemon, myTeam, setMyTeam }) {
 
   function renderPokemonCardFactory() {
-    return pokemon.map((singlePokemon, indexOfSinglePokemon) => (
+    return pokemon.map((singlePokemon) => (
       <Card
-        key={singlePokemon.name + indexOfSinglePokemon}
+        key={singlePokemon.id}
```

Comment:
Using the index of an array item for the `key` prop value is not recommended because the index of an item within the array might change if the elements are filtered or sorted. Each Pokemon item has a unique `id` property value, so I switched to using that for the `key`.